cmd/rancher-desktop-guestagent/main.go

```diff
@@ -181,7 +181,7 @@ func main() {
 			k8sServiceListenerIP := net.ParseIP(*k8sServiceListenerAddr)
 
 			if k8sServiceListenerIP == nil || !(k8sServiceListenerIP.Equal(net.IPv4zero) ||
-				k8sServiceListenerIP.Equal(net.IPv4(127, 0, 0, 1))) { //nolint:gomnd // IPv4 addr localhost
+				k8sServiceListenerIP.Equal(net.IPv4(127, 0, 0, 1))) { //nolint:gomnd,mnd // IPv4 addr localhost
```

Comment:
It might be better to update the lint config to allow magic numbers for `net.IPv4` instead.